2001-3000/LIT2900RepCh181.xml

```diff
@@ -1,73 +1,484 @@
-                ሰላም፡ ለዝክረ፡ ስምኪ፡ ዘኆኅያቲሁ፡ አርባዕ።
-               </ab>
-            </div>
-         </div>
-      </body>
-   </text>
-</TEI>
\ No newline at end of file
+  <teiHeader xml:base="https://betamasaheft.eu/">
+    <fileDesc>
+        <titleStmt>
+          <title xml:lang="gez" xml:id="t1">መልክአ፡ አርሲማ፡</title>
+          <title xml:lang="gez" corresp="#t1" type="normalized">Malkəʾa ʾArsimā</title>
+          <title xml:lang="en" corresp="#t1">Image of Hripsime</title>
+
+          <editor role="generalEditor" key="AB"/>
+          <editor key="AD"/>
+
+          <funder>Akademie der Wissenschaften in Hamburg</funder>
+        </titleStmt>
+        <publicationStmt>
+          <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+          <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale Forschungsumgebung / Beta maṣāḥǝft</publisher>
+          <pubPlace>Hamburg</pubPlace>
+          <availability>
+              <licence target="http://creativecommons.org/licenses/by-sa/4.0/">
+                <p>This file is licensed under the Creative Commons Attribution-ShareAlike 4.0.</p>
+              </licence>
+          </availability>
+        </publicationStmt>
+        <sourceDesc>
+          <p>Work of the literatures of Ethiopia and Eritrea</p>
+        </sourceDesc>
+        <sourceDesc>
+          <listWit>
+            <witness corresp="BLorient577" xml:id="A"/>
+            <witness corresp="EMML6993" xml:id="B"/>
+        </listWit>
+        </sourceDesc>
+    </fileDesc>
+    <encodingDesc>
+      <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="https://raw.githubusercontent.com/BetaMasaheft/Documentation/master/prefixDef.xml">
+        <xi:fallback>
+          <p>Definitions of prefixes used.</p>
+        </xi:fallback>
+      </xi:include>
+    </encodingDesc>
+    <profileDesc>
+        <langUsage><language ident="en">English</language><language ident="gez">Gǝʿǝz</language></langUsage>
+        <textClass>
+          <keywords scheme="#ethioauthlist">
+              <term key="Poetry"/>        
+              <term key="ChristianLiterature"/>
+              <term key="Hagiography"/>
+          </keywords>
+        </textClass>
+    </profileDesc>
+    <revisionDesc>
+        <change who="PL" when="2016-07-26">Created XML record from Ethio authority google spreadsheet</change>
+        <change who="MV" when="2017-06-01">Added keywords and incipit</change>
+        <change who="AD" when="2024-05-13">Added encoded critical edition and translation</change>
+    </revisionDesc>
+  </teiHeader>
+  <text xml:base="https://betamasaheft.eu/">
+    <body>
+
+      <div type="bibliography">
+        <listBibl type="editions">
+          <bibl>
+            <ptr target="bm:Chaine1913Rep"/>
+            <citedRange unit="item">181</citedRange>
+          </bibl>
+
+          <bibl>
+            <ptr target="bm:Dickinson2023Arsima"/>
+          </bibl>
+        </listBibl>
+      </div>
```

Comment:
I just thought that one could also add the relation 
```
            <listRelation>
                <relation name="ecrm:P129_is_about" active="LIT2900RepCh181" passive="PRS8126Rhipsime"/>
             </listRelation>
```

Reply:
and one could also add
```
<listBibl type="translation">
          <bibl>
            <ptr target="bm:Dickinson2023Arsima"/>
          </bibl>
        </listBibl>
```

Reply:
Both came to my mind at one point or another then I promptly forgot about them — let me add them.